src/librustc_metadata/schema.rs

```diff
@@ -207,6 +207,7 @@ pub struct CrateRoot {
     pub impls: LazySeq<TraitImpls>,
     pub exported_symbols: EncodedExportedSymbols,
     pub wasm_custom_sections: LazySeq<DefIndex>,
+    pub interpret_alloc_index: Vec<u32>,
```

Comment:
I'm seeing ICEs inside the deserialize impl for this field in 4 stage 2 run-pass tests with aux builds. I can't reproduce on stage 1 and Travis also takes it. Maybe there's some stale data where crates are loaded that were compiled without this field?

Also: should this be LazySeq?

Reply:
If you have `ignore_git` set (which is the default now), incompatible metadata won't be detected. I think I had this issue with test auxiliary crates. And yes, please make it a `LazySeq<u32>`.

Reply:
I changed it into a `LazySeq`, now I'm seeing the failures on stage 1, too. `ignore_git` doesn't change anything

Reply:
The failure seems to be in the deserialization in the field after `interpret_alloc_index`.